content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Remove deprecated API resources"
+linkTitle: "Remove deprecated API resources"
+weight: 20
+type: "docs"
+---
+
+We've deprecated the following `cert-manager` APIs: 
+- `cert-manager.io/v1alpha2`
+- `cert-manager.io/v1alpha3`
+- `cert-manager.io/v1beta1`
+- `acme.cert-manager.io/v1alpha2`
+- `acme.cert-manager.io/v1alpha3`
+- `acme.cert-manager.io/v1beta1`
+These APIs will be removed in "cert-manager" `v1.6.0`.
```

Comment:
suggestion: sorry, I should've been clearer: I meant we could replace `cert-manager` with cert-manager on its own. Like this:

```suggestion
These APIs will be removed in cert-manager `v1.6.0`.
```